src/subcommand/server.rs

```diff
@@ -887,6 +888,56 @@ impl Server {
     })
   }
 
+  async fn address_cardinals(
+    Extension(server_config): Extension<Arc<ServerConfig>>,
+    Extension(index): Extension<Arc<Index>>,
+    Path(address): Path<Address<NetworkUnchecked>>,
+    AcceptJson(accept_json): AcceptJson,
+  ) -> ServerResult {
+    task::block_in_place(|| {
+      Ok(if accept_json {
+        let address = address
+          .require_network(server_config.chain.network())
+          .map_err(|err| ServerError::BadRequest(err.to_string()))?;
+
+        let mut outputs = index.get_address_info(&address)?;
+
+        outputs.sort();
```

Comment:
Do people rely on lexicographical ordering? Or should it be ordered by size?

Reply:
I wasn't sure either! But followed the precedence of `address`

https://github.com/ordinals/ord/blob/f1c1ee5adb39c54e99c79e65791b6339186c35be/src/subcommand/server.rs#L860